src/common-lib/urls/getRelativeConfigURLs.js

```diff
@@ -0,0 +1,45 @@
+// URLs should end with a `/` to avoid redirects from
+// e.g. `/unit` to `/unit/index.html` during tests.
+
+function getRelativeConfigURLs() {
+  return [
+    // Error pages
+    "/404",
+    // Public pages
+    "/",
+    "/lesson-planning",
+    "/develop-your-curriculum",
+    "/support-your-team",
+    "/contact-us",
+    "/about-us/who-we-are",
+    "/about-us/leadership",
+    "/about-us/board",
+    "/about-us/partners",
+    "/about-us/work-with-us",
+    "/blog",
+    "/blog/how-to-design-a-unit-of-study",
+    "/blog/evolution-of-oak",
+    "/blog/join-the-childrens-mental-health-week-assembly-2022",
+    "/webinars/effective-feedback-a-practical-guide-to-using-feedback-to-enhance-student",
+    "/legal/accessibility-statement",
+    // landing pages
```

Comment:
Could we have a note in the code to say why some of the URLs are commented out, with a link to the ticket on GH (not ZH, since we may not keep it)?